home/templates/home/base.html

```diff
@@ -3,61 +3,40 @@
 {% load static %}
 
 <head>
-    <!-- Required meta tags -->
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="description"
-          content="Penny University is a peer learning group with the goal of connecting people in order to share knowledge. Together, we learn!">
-
-    <!-- Bootstrap CSS -->
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
-          integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+          content="Penny University is a peer-to-peer learning community.
+          We connect people who enjoy working together to gain knowledge and improve their skills.">
 
     {% load static %}
-    <link rel="stylesheet" type="text/css" href="{% static 'home/style.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static 'css/style.css' %}">
     <link rel="shortcut icon" type="image/x-icon" href="{% static 'home/favicon.ico' %}">
     <link rel="icon" type="image/x-icon" href="{% static 'home/favicon.ico' %}">
 
     <!-- Add icon library -->
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 
-    <style>
-        html, body {
-            height: 100%;
-        }
-
-        .overlay-text {
-            position: relative;
-            z-index: 1;
-        }
-
-        .postion-relative {
-            position: relative;
-        }
+    <!-- Add fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700|Roboto+Slab&display=swap" rel="stylesheet">
 
+    <style>
```

Comment:
Feels like we should get this last style block out of this file - especially since it deals with something that is only used on the index page. Do you have any recommendations?

Reply:
Maybe in style.scss ?